packages/xod-client-electron/src/view/containers/App.jsx

```diff
@@ -332,6 +332,10 @@ class App extends client.App {
     );
   }
 
+  onExitClicked() {
+    let w = remoteElectron.getCurrentWindow().close();
```

Comment:
Not sure why `let w`, though

Reply:
Yeah, copy-paste blindness. ESLint barked at me. Fixed.